### PR TITLE
added svg loader for webpack

### PR DESCRIPTION
**Pull request by koskinpark**
Closes https://github.com/skilld-labs/kaizen/issues/56

#Usage 
from twig:
```
<svg aria-hidden="true">
  <use xlink:href="{{ kaizen_svg_sprite_path }}#svg-test"></use>
</svg>
```

from js:
`import getSvgIcon from './svg-sprite';`
`document.body.innerHTML += getSvgIcon('test');`

### CODE REVIEW

File: STARTERKIT/basic/kaizen.theme [hunk 1]
```diff
@@ -1,11 +1,26 @@
 <?php
 
+/**
+ * Returns path to svg sprite.
+ */
+function kaizen_get_svg_sprite_path() {
+  return base_path() . \Drupal::theme()->getActiveTheme()->getPath() . '/dist/svg/sprite.svg';
```
Comment (andypost) [inline STARTERKIT/basic/kaizen.theme]: are you sure this should point to active theme? probably it related only to kaizen and its child themes

File: STARTERKIT/basic/kaizen.theme [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+}
+
+/**
+ * hook_page_attachments().
+ *
+ * Prepares svg_sprite_path variable to get full path to svg sprite from js.
+ */
+function kaizen_page_attachments_alter(array &$attachments) {
+  $attachments['#attached']['drupalSettings']['svg_sprite_path'] = test_get_svg_sprite_path();
```
Comment (andypost) [inline STARTERKIT/basic/kaizen.theme]: the function name needs work `test_get_svg_sprite_path()` is no-go

Comment (andypost) [reply STARTERKIT/basic/kaizen.theme]: I mean it should be prefixed with theme name, a-la `_kaizen_get_svg_sprite_path`

Comment (koskinpark) [reply STARTERKIT/basic/kaizen.theme]: oh, forgot to rename, true

**Comment by iberdinsky-skilld** [comment]
rebased on master

**Comment by iberdinsky-skilld** [comment]
removed icons task. seems all fine for mvp. creating issue some features request which may be useful.

File: STARTERKIT/basic/kaizen.theme (hunk 1 above)
Comment (iberdinsky-skilld) [reply STARTERKIT/basic/kaizen.theme]: created follow up for this -> #85

**Pull request merged**